lib/elixir/lib/version.ex

```diff
@@ -24,8 +25,8 @@ defmodule Version do
 
   ## Struct
 
-  The version is represented by the `Version` struct and fields
-  are named according to SemVer: `:major`, `:minor`, `:patch`,
+  The version is represented by the [`%Version{}`](`t:Version.t/0`) struct and fields
+  are named according to [SemVer](http://semver.org/): `:major`, `:minor`, `:patch`,
```

Comment:
We don't have to link to the semver website multiple times, we already have a link on line 10.